### DISCUSSION TITLE
我的nonebot无法启动

### DISCUSSION THREAD

**Discussion by Dusky-Ink** [nonebot/discussions#358] [问答]
在运行nb run之后出现如下界面
![H32~SWNWEK_I)DI~ZUTD1J1](https://user-images.githubusercontent.com/113438889/228265680-f01c5ee3-81d3-4189-a2bd-5dedadf76251.png)
![N在运行nb run之后出现如下界面
![H32~SWNWEK_I)DI~ZUTD1J1](https://user-images.githubusercontent.com/113438889/228265680-f01c5ee3-81d3-4189-a2bd-5dedadf76251.png)
7TSG3H0WHERK9NU)%%)2](https://user-images.githubusercontent.com/113438889/228265720-6017706a-a57c-4721-adca-ef910e958d52.png)

**Comment by yanyongyu** [comment]
crazy thursday插件报错，请去对应插件提问

**Comment by KafCoppelia** [comment]
[`v0.2.6.post1`](https://github.com/MinatoAquaCrews/nonebot_plugin_crazy_thursday/releases/tag/v0.2.6.post1) 已修复
